corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -160,7 +160,11 @@
           <!--/ko-->
         </div>
         <h4 class="panel-release-title">
-          <strong>{% trans "Version" %} <span data-bind="text: version"></span></strong> |
+          <strong>{% trans "Version" %} <span data-bind="text: version"></span></strong>
+          <span data-bind="visible: doc_type() == 'LinkedApplication'">
+            (master v<span data-bind="text: pulled_from_master_version"></span>)
```

Comment:
i am not sure if space is a constraint.
this would be clearer with `pulled from master v` instead.

Reply:
Leaving this as-is, because of space. There's going to be even more info crammed in here when it also needs to note which master app was pulled.